samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,99 @@ nRF5340: Network core bootloader
    :local:
    :depth: 2
 
-This bootloader sample implements an immutable first stage bootloader that has the capability to update the application firmware on the network core of the nRF5340 System on Chip (SoC).
-In addition to this, the network core bootloader sample performs flash protection of both itself and the application.
+The network core bootloader sample implements an immutable first-stage bootloader that can update the application firmware on the network core of the nRF5340 System on Chip (SoC).
 
 Overview
 ********
 
-The network core bootloader sample supports transporting an already verified and authenticated firmware upgrade from the application core flash to the network core flash, as well as performing flash protection.
+This sample supports transporting an already verified and authenticated firmware update from the application core flash memory to the network core flash memory.
+Also, this sample protects the flash memory areas allocated to both itself and the application running on the network core.
 
-This is accomplished by the following steps:
+The bootloader accomplishes these objectives as follows:
```

Comment:
```suggestion
The bootloader accomplishes the following objectives:
```

Reply:
What I meant here is that the 2 previously listed objectives are accomplished by the following sequence of actions. I'll rephrase it.

Reply:
I rephrased the entire section a bit, the reason for line 18 should now be more clear.